packages/api/amplify_api_ios/ios/Classes/SwiftAmplifyApiPlugin.swift

```diff
@@ -21,63 +21,25 @@ import amplify_flutter_ios
 import AWSPluginsCore
 
 public class SwiftAmplifyApiPlugin: NSObject, FlutterPlugin, NativeApiBridge {
-    private let bridge: ApiBridge
-    private let graphQLSubscriptionsStreamHandler: GraphQLSubscriptionsStreamHandler
     static var methodChannel: FlutterMethodChannel!
 
-    init(
-        bridge: ApiBridge = ApiBridge(),
-        graphQLSubscriptionsStreamHandler: GraphQLSubscriptionsStreamHandler = GraphQLSubscriptionsStreamHandler()
-    ) {
-        self.bridge = bridge
-        self.graphQLSubscriptionsStreamHandler = graphQLSubscriptionsStreamHandler
-    }
-
     public static func register(with registrar: FlutterPluginRegistrar) {
         methodChannel = FlutterMethodChannel(
```

Comment:
callout: I wasn't totally sure if this part of method channel was still needed? I was thinking we still needed it for oidc/lambda communication over the channel related to getting token.